@remirror/extension-tables/package.json

```diff
@@ -16,7 +16,6 @@
   "module": "lib/dist/extension-tables.esm.js",
   "types": "lib/index.d.ts",
   "dependencies": {
-    "@babel/runtime": "^7",
```

Comment:
Ooops, this one is needed. It's one of the implicit dependencies since the babel build relies on it being present. I forgot to mention.